dll/win32/shell32/folders/CDesktopFolder.cpp

```diff
@@ -1067,37 +1067,25 @@ HRESULT WINAPI CDesktopFolder::GetCurFolder(PIDLIST_ABSOLUTE * pidl)
 
 HRESULT WINAPI CDesktopFolder::CallBack(IShellFolder *psf, HWND hwndOwner, IDataObject *pdtobj, UINT uMsg, WPARAM wParam, LPARAM lParam)
 {
-    if (uMsg != DFM_MERGECONTEXTMENU && uMsg != DFM_INVOKECOMMAND)
-        return S_OK;
-
+    enum { IDC_PROPERTIES };
```

Comment:
Is it, in a sense, the "same" properties menu-item command as for the drives and FS folders? I was wondering whether it could be defined in some shell32 header :thinking:

Reply:
Right now it is the same but it really belongs inside that callback. If you add more menu items you would do `enum { IDC_PROPERTIES, IDC_SOMETHINGELSE };` and those ids only apply to DFM_MERGECONTEXTMENU and DFM_INVOKECOMMAND inside that particular callback function. Ideally all those menu ids should start at 0 so it does not really make sense to try to make those global IMHO.